src/test/starknet.t.sol

```diff
@@ -20,29 +20,30 @@ import "../DssSpell.t.base.sol";
 
 contract ConfigStarknet {
     StarknetValues starknetValues;
-    bytes32 l2Spell;
 
     struct StarknetValues {
+        bytes32 l2_spell;
         address core_implementation;
         uint256 dai_bridge_isOpen;
         uint256 dai_bridge_ceiling;
         uint256 dai_bridge_maxDeposit;
         uint256 l2_dai_bridge;
         uint256 l2_gov_relay;
+        uint256 relay_selector;
     }
 
     function setValues() public {
         uint256 WAD = 10 ** 18;
 
-        l2Spell = 0x00a052591661d7e249b46a1084c63b14dae6aa8b1a56ab3f7df8c8add1c374b1;  // Set to zero if no spell is set.
-
         starknetValues = StarknetValues({
-            core_implementation:       0x60C5fA1763cC9CB9c7c25458C6cDDFbc8F125256,
+            l2_spell:                  0x00a052591661d7e249b46a1084c63b14dae6aa8b1a56ab3f7df8c8add1c374b1,  // Set to zero if no spell is set.
+            core_implementation:       0x70c8A579aD08339cCA19d77d8646F4b6f0fd098A,
             dai_bridge_isOpen:         1,                     // 1 open, 0 closed
             dai_bridge_ceiling:        1_000_000 * WAD,       // wei
             dai_bridge_maxDeposit:     type(uint256).max,     // wei
             l2_dai_bridge:             0x057b7fe4e59d295de5e7955c373023514ede5b972e872e9aa5dcdf563f5cfacb,
-            l2_gov_relay:              0x00275e3f018f7884f449a1fb418b6b1de77e01c74a9fefaed1599cb22322ff74
+            l2_gov_relay:              0x00275e3f018f7884f449a1fb418b6b1de77e01c74a9fefaed1599cb22322ff74,
+            relay_selector:            300224956480472355485152391090755024345070441743081995053718200325371913697
```

Comment:
please move here also the previous clarification that             this is hard-coded in L1 gov relay and not public.

Reply:
I see it's done so resolving.